### PR TITLE
2592 UI play screen

### PR DESCRIPTION
**Pull request by chicky-dev**
https://edu-redmine.framgia.vn/issues/2592

### CODE REVIEW

**Comment by chicky-dev** [comment]
ready

File: app/src/main/res/drawable/gradient_to_white.xml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android">
+    <gradient
+        android:startColor="@android:color/white"
+        android:endColor="@color/color_blue_gray"
+        android:angle="-90" />
+</shape>
```
Comment (NguyenMinhTuanB) [inline app/src/main/res/drawable/gradient_to_white.xml]: need an endline

File: app/src/main/res/layout/layout_play_info.xml [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:src="@drawable/ic_info"
+        app:layout_constraintBottom_toBottomOf="@id/image_download"
+        app:layout_constraintLeft_toRightOf="@id/image_download"
+        app:layout_constraintRight_toLeftOf="@id/image_favourite"
+        app:layout_constraintTop_toTopOf="@id/image_download" />
+
+    <ImageView
+        android:id="@+id/image_favourite"
```
Comment (NguyenMinhTuanB) [inline app/src/main/res/layout/layout_play_info.xml:47]: favorite

File: app/src/main/res/drawable/ic_favourite.xml [hunk 3]
```diff
@@ -0,0 +1,4 @@
+<vector android:height="24dp" android:viewportHeight="268.476"
```
Comment (NguyenMinhTuanB) [inline app/src/main/res/drawable/ic_favourite.xml]: wrong name. ic_favorite

**Comment by NguyenMinhTuanB** [comment]
commented

**Comment by chicky-dev** [comment]
ready

**Review by NguyenMinhTuanB — APPROVED** [review]

**Comment by NguyenMinhTuanB** [comment]
merged

**Pull request merged**